app/src/main/kotlin/com/softteco/template/navigation/AppNavHost.kt

```diff
@@ -66,6 +67,13 @@ fun NavGraphBuilder.bottomBarGraph(navController: NavController) {
         composable(Screen.SignUp.route) {
             SignUpScreen(onBackClicked = { navController.navigateUp() })
         }
+        composable(route = Screen.ResetPassword.route) {
+            val token = "token" // TODO: deep link
+            ResetPasswordScreen(
+                gotToLoginScreen = { navController.navigate(Screen.Login.route) },
+                token = token
```

Comment:
It would be better to get the argument directly in the viewModel, rather than passing it as a composable argument (https://developer.android.com/jetpack/compose/navigation#nav-with-args)